components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2823,15 +2823,27 @@ pub struct CatchUpLogs {
 
 pub struct GenSnapTask {
     pub(crate) region_id: u64,
+    // Fill it after the RocksDB snapshot is taken.
+    pub index: Arc<AtomicU64>,
```

Comment:
Seems that `index` doesn't need to be `atomic`.

Reply:
It's filled in region worker, but fetched in raftstore thread. So I guess `Arc` is the most simple way to deal with it.

Reply:
I get it. BTW, it is filled in apply thread not region worker.